### ISSUE TITLE
Join/Parts are displayed twice

### ISSUE THREAD

**Issue by Ppjet6** [42wim/matterircd#104]
## Steps to reproduce

- Connect to mattermost via matterircd
- Have somebody else connect to mattermost (or matterircd for that matter)

## What happens

The new person joining is displayed as (on irssi):
```
-!- XXX [foo@bar] has joined #baz 
< XXX> XXX has joined the channel.
```

## What I expect

Only the `-!- XXX [foo@bar] has joined #baz` line to be displayed.
I don't know if this is matterircd or a protocol limitation in mattermost though.

EDIT: The same happens for parts

**Comment by 42wim** [comment]
No, that's a cosmetic bug ;-)

**Comment by 42wim** [comment]
Fixed in master, release v0.12.1 soon
